Makefile

```diff
@@ -493,6 +493,8 @@ install.bin-nobuild:
 	install ${SELINUXOPT} -d -m 755 $(DESTDIR)$(BINDIR)
 	install ${SELINUXOPT} -m 755 bin/podman $(DESTDIR)$(BINDIR)/podman
 	test -z "${SELINUXOPT}" || chcon --verbose --reference=$(DESTDIR)$(BINDIR)/podman bin/podman
+	install ${SELINUXOPT} -m 755 -d ${DESTDIR}${TMPFILESDIR}
```

Comment:
For historical purposes, could you add a comment here explaining this?

Reply:
You want a comment in the man page explaining why we are installing podman.conf, or do you want a comment in podman.conf telling you what the line in their is for?  Telling systemd to not prune /tmp/podman-run-*?

Reply:
I was thinking of just a comment here explaining the change with a pointer to the issue.  If you think a man page change would serve better, and/or a comment in `/tmp/podman-run-*` I'm fine with either or both.  Just so that two years from now someone doesn't remove it without realizing it's need.

Reply:
I am just thinking the message should go into the podman.conf file.  Not here.  If I am removing content, I am likely to read the file.  Added explanation to podman.conf file.